apps/src/lib/kits/maker/boards/microBit/MBFirmataUpdater.js

```diff
@@ -28,6 +30,10 @@ export default class MBFirmataUpdater {
     } else if (MICROBIT_IDS_V2.includes(microBitId)) {
       microBitVersion = MICROBIT_V2;
     }
+    analyticsReporter.sendEvent(EVENTS.MAKER_SETUP_PAGE_MB_VERSION_EVENT, {
+      'micro:bit version': microBitVersion
```

Comment:
I am very slightly concerned about a colon in a key. I think it'll be okay but this is the kind of thing that can lead to parsing errors. I'd recommend testing this PR on staging after its merged (staging actually logs to Amplitude) to double check there aren't any issues

Reply:
It's definitely not a problem to remove the colon so will remove - thanks for the heads up on that. I also am about to post a question in Slack #teacher-tools about being able to test before merging to staging. Thank you!

Reply:
And thanks for sharing this [Amplitude for Developers](https://docs.google.com/document/d/1uuo-vJs41SvWRokVHGu8trZr-h6axq6BHqe6fbjlRbk/edit#) via Slack. I added a unit test for analytics based on the doc suggestion.